class.geocoder.php

```diff
@@ -167,10 +167,15 @@ private function osm_geocode ( $address ) {
         $json = $this->get_url($geocode_url);
         $json = json_decode($json);
         
-        return (Object) array(
-            'lat' => $json[0]->lat,
-            'lng' => $json[0]->lon,
-        );
+        if (isset($json[0]->lat) && isset($json[0]->lon)) {
+            return (Object) array(
+                'lat' => $json[0]->lat,
+                'lng' => $json[0]->lon,
+            );
+        } else {
+            throw new Exception('No Address Found.');
```

Comment:
I tested it more. There is no exception. It returns false and the default lan and lng are displayed.
This line could be deleted.